strings/tips.txt

```diff
@@ -186,3 +186,4 @@ As a Ghost, you can double click on people, bots, or the singularity to follow t
 As a Devil, you gain power for every three souls you control, however you also become more obvious.
 As a Devil, as long as you control at least one other soul, you will automatically resurrect, as long as a banishment ritual is not performed.
 At which time a Devil's nameth is spake on the tongue of man, the Devil may appeareth. 
+As a Security Officer, remember that causation does not equal correlation. Someone may have just been at the wrong place at the wrong time!
```

Comment:
other way around, causation is a correlation but not all correlations imply causation